src/Illuminate/Support/Manager.php

```diff
@@ -1,6 +1,7 @@
 <?php namespace Illuminate\Support;
 
 use Closure;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
Please order these by length, not alphabetically.

And squash to one commit when you've fixed this.